### PR TITLE
feat(projects): add project access token api

### PR DESCRIPTION
**Pull request by max-wittig**
Fun Trivia: The **create** endpoint is totally useless btw. as it doesn't return the token and there is no way to get it

https://gitlab.com/gitlab-org/gitlab/-/issues/238991#note_516433443

### CODE REVIEW

File: gitlab/tests/objects/test_project_access_tokens.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        rsps.add(
+            method=responses.GET,
+            url="http://localhost/api/v4/projects/1/access_tokens",
+            json=content,
+            content_type="application/json",
+            status=200,
+        )
+        yield rsps
+
+
```
Comment (nejch) [inline gitlab/tests/objects/test_project_access_tokens.py:117]: Looks like this is already on lines 9-32 :smile: 
```suggestion
```

**Review by nejch — CHANGES_REQUESTED** [review]
Nice! Just found one little duplicate.

Also lol :facepalm: I had high hopes for this endpoint in the release notes but looks pointless to me :(

File: gitlab/tests/objects/test_project_access_tokens.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
```
Comment (nejch) [inline gitlab/tests/objects/test_project_access_tokens.py:84]:
```suggestion
```

Comment (max-wittig) [reply gitlab/tests/objects/test_project_access_tokens.py:84]: What did I do 😄

**Pull request merged**